sdks/python/test-suites/dataflow/common.gradle

```diff
@@ -109,4 +109,21 @@ task validatesRunnerStreamingTests {
       args '-c', ". ${envdir}/bin/activate && ${runScriptsDir}/run_integration_test.sh $cmdArgs"
     }
   }
-}
\ No newline at end of file
+}
+
+task runPerformanceTest {
+    dependsOn 'installGcpTest'
+    dependsOn ':sdks:python:sdist'
+
+    def test = project.findProperty('test')
+    def testOpts = project.findProperty('test-pipeline-options')
+    testOpts += " --sdk_location=${files(configurations.distTarBall.files).singleFile}"
+
+  doLast {
+    exec {
+      workingDir "${project.rootDir}/sdks/python"
+      executable 'sh'
+      args '-c', ". ${envdir}/bin/activate && ${envdir}/bin/python setup.py nosetests --tests=${test}  --test-pipeline-options=\"${testOpts}\" --ignore-files \'.*py3\\d?\\.py\$\'"
```

Comment:
Do we need to pass `--ignore-files` given that we control which tests to run?

Reply:
I wanted to be on the safe side because in most places (not only in the bash scripts) it is added. But when I think about it we absolutely don't need to ignore anything, we run just one test. I'll remove it.